docs/conf.py

```diff
@@ -3,17 +3,16 @@
 # Ref: https://www.sphinx-doc.org/en/master/usage/configuration.html
 """Configuration file for the Sphinx docs."""
 
-from functools import partial
+import os
+
+from importlib.metadata import version as get_version
 from pathlib import Path
 from sys import path
 
-from setuptools_scm import get_version
```

Comment:
What is the reason you don't want to keep this? I think it's rather reasonable to have this as a docs dependency.